ports/gflags/vcpkg-cmake-wrapper.cmake

```
@@ -4,6 +4,7 @@ _find_package(${ARGS})
 
 foreach(tgt gflags gflags_shared gflags_static)
     if (NOT TARGET ${tgt} AND TARGET "gflags::${tgt}")
+        set_target_properties("gflags::${tgt}" PROPERTIES IMPORTED_GLOBAL TRUE)
         add_library(${tgt} ALIAS "gflags::${tgt}")

```

Probably something like:
```suggestion
        add_library(${tgt} INTERFACE IMPORTED UNKNOWN)
        target_link_libraries(${tgt} INTERFACE "gflags::${tgt}")
```
is better since it does not require to make the targets global.